frontend/packages/dev-console/src/components/pipelineruns/PipelineTaskLogs.scss

```diff
@@ -0,0 +1,41 @@
+.odc-pipeline-task-logs{
+  &__container {
+  margin-top: var(--pf-global--spacer--lg);
+  background-color: #030303;
```

Comment:
No pf variable for this?

Reply:
@divyanshiGupta I was looking for a pf-global variable (not pf-chart) corresponding to #030303 couldn't find any and we need to keep the background uniform with the `ResourceLog` component.

Reply:
If you cant find a pf variable for this then it's okay to use the color code